frontend/src/pages/404/index.jsx

```diff
@@ -19,11 +22,11 @@ function NotFound() {
         </div>
         <div className="col">
           <h1 className="mb-4">{tPNF('title')}</h1>
-          <p className="fs-5 text-black mb-3">{tPNF('whatHappened.title')}</p>
```

Comment:
@sergiozeppo i watched code and current behavior and consider that we can just remove `text-black` color classes and do not add another. It will work well on dark and white theme.

BTW bootstrap has class for default color https://getbootstrap.com/docs/5.3/utilities/colors/  `.text-body`

Reply:
> i watched code and current behavior and consider that we can just remove `text-black` color classes and do not add another. It will work well on dark and white theme.

Hi there, @fey ! You was totally right, deleted all of this unnecessary stuff. Works as intended. 

> BTW bootstrap has class for default color https://getbootstrap.com/docs/5.3/utilities/colors/ `.text-body`

Thanks for remark!